docs/adr/0011-brighter-support-for-bulk-messaging-operations.md

```diff
@@ -13,26 +13,26 @@ To support Transactional Messaging the Command Processor uses an Outbox when pro
 * A `DepositPost` call to translate the `IRequest` into a `Message` and store it in an `Outbox`
 * A `ClearOutbox` call to retrieve the `Message` from the `Outbox` and dispatch it via a `Producer`
 
-Within these two steps we peform a number of activities.
+Within these two steps we perform a number of activities.
 
 For `DepositPost` we:
 
-* Lookup the `IAmAProducer` associated with a `Request
-* Translate an `IRequest` into the `Message`, excuting the `IAmAMessageTransform` and `IAmAMessagemapper` pipeline
+* Translate an `IRequest` into the `Message`, executing the `IAmAMessageTransform` and `IAmAMessageMapper` pipeline
```

Comment:
I think we look up the producer now, in order to add the Publication to the pipeline

Reply:
I've added this back in